lib/okta.go

```diff
@@ -578,10 +579,19 @@ func (o *OktaClient) Get(method string, path string, data []byte, recv interface
 		}
 	}
 
+       // FORCE TLS negotiation to only support HTTP/1.1
+       tlsCfg := &tls.Config{
+                NextProtos: []string{"h1"},
+       }
+
 	transCfg := &http.Transport{
 		Proxy:               http.ProxyFromEnvironment,
 		TLSHandshakeTimeout: Timeout,
+                DisableKeepAlives:   true,
+                MaxIdleConnsPerHost: -1,
```

Comment:
are these two settings also necessary?

Reply:
These weren't required in our repro. `DisableKeepAlives` makes the client setup a new TLS connection for each request. Though, that's the behavior I'm already seeing without it explicitly set.